Project.toml

```diff
@@ -4,7 +4,9 @@ authors = ["Invenia Technical Computing"]
 version = "0.4.0"
 
 [deps]
+Distances = "b4f34e82-e78d-54a5-968a-f98e89d6e8f7"
 IterTools = "c8e1da08-722c-5040-9ed9-7db0dc04731e"
+NearestNeighbors = "b8a86587-4115-5ab1-83bc-aa920d37bbce"
```

Comment:
Nice, I didn't realize that NearestNeighbors.jl was such a light dependency 👍 . That's been my main reluctance for adding more sophisticated methods that should depend on external packages.